src/TestUtils/src/DeviceTests/AssertionExtensions.cs

```diff
@@ -42,5 +42,14 @@ public static void AssertWithMessage(Action assertion, string message)
 				Assert.True(false, $"Message: {message} Failure: {e}");
 			}
 		}
+
+		public static void CloseEnough(double expected, double actual, double epsilon = 0.2, string? message = null)
```

Comment:
Why is it that we can't get equal but only "close enough" with the various flyout frame measurements?

Reply:
The conversion of our DP values to pixels based on the density of the device and then back to DP will result in some fractional precision loss. A 0.2 epsilon represents less than half a pixel.